src/containers/Application/ApplicationHome.tsx

```diff
@@ -1,21 +1,86 @@
-import React from 'react'
-import { FullStructure } from '../../utils/types'
+import React, { useEffect } from 'react'
+import { Button, Divider, Header, Message, Segment, Sticky } from 'semantic-ui-react'
+import { FullStructure, SectionAndPage, StageAndStatus, TemplateDetails } from '../../utils/types'
 import useGetFullApplicationStructure from '../../utils/hooks/useGetFullApplicationStructure'
+import { ApplicationHeader, Loading } from '../../components'
+import strings from '../../utils/constants'
+import { useUserState } from '../../contexts/UserState'
+import SectionsProgress from '../../components/Sections/SectionsProgress'
+import { useRouter } from '../../utils/hooks/useRouter'
+import { ApplicationStatus } from '../../utils/generated/graphql'
 
 interface ApplicationProps {
   structure: FullStructure
+  template: TemplateDetails
 }
 
-const ApplicationHome: React.FC<ApplicationProps> = ({ structure }) => {
+const ApplicationHome: React.FC<ApplicationProps> = ({ structure, template }) => {
+  const {
+    query: { serialNumber },
+    push,
+  } = useRouter()
+  const {
+    userState: { currentUser },
+  } = useUserState()
+
   const { error, fullStructure } = useGetFullApplicationStructure({
     structure,
   })
 
-  console.log('FULL STRUCTURE', fullStructure)
-  console.log('responsesByCode', fullStructure?.responsesByCode)
-  console.log('isLoading', !fullStructure)
+  useEffect(() => {
+    if (!fullStructure) return
+    const { status } = fullStructure.info.current as StageAndStatus
+    if (status !== ApplicationStatus.Draft && status !== ApplicationStatus.ChangesRequired)
+      push(`/application/${serialNumber}/summary`)
```

Comment:
this should be replace, otherwise back button will always jump back to home then to summary

Reply:
I found that "replace" made going back impossible, since the previous URL was wiped out (or maybe I was just using it wrong).

Perhaps the Router `.go(2)` method might be required, or perhaps play round with options until we're sure we get the intended behaviour. (https://reactrouter.com/web/api/history)

Reply:
Cool. Thanks changing this now.

Reply:
Leaving this as `push` for now. If another issue is required to decide and make this change I think I can be added to the [Epic#10](https://github.com/openmsupply/application-manager-web-app/issues/78)